cli/replicaset/cmd/status.go

```diff
@@ -73,9 +75,9 @@ func statusReplicasets(replicasets replicaset.Replicasets) error {
 	return nil
 }
 
-// fillAliases fills missed aliases with UUID. The case: Tarantool 1.10 without
+// FillAliases fills missed aliases with UUID. The case: Tarantool 1.10 without
 // an orchestrator.
-func fillAliases(replicasets replicaset.Replicasets) replicaset.Replicasets {
+func FillAliases(replicasets replicaset.Replicasets) replicaset.Replicasets {
```

Comment:
We need a unit-test for the method since it is public at now.

Reply:
It's private again.